Sources/GRPC/Shims.swift

```diff
@@ -47,3 +47,15 @@ extension GRPCStatusTransformable {
     return self.makeGRPCStatus()
   }
 }
+
+extension GRPCClient {
+  @available(*, deprecated, renamed: "init(channel:defaultCallOptions:)", message: "You may also need to regenerate your client code")
+  public init(connection: ClientConnection, defaultCallOptions: CallOptions = CallOptions()) {
```

Comment:
I assume we will delete these before 1.0?

Reply:
That's right, this whole file will be removed (there's a TODO at the top of it)